src/main/java/baritone/process/CustomGoalProcess.java

```diff
@@ -74,8 +77,48 @@ public boolean isActive() {
         return this.state != State.NONE;
     }
 
+    /*
```

Comment:
Maybe you wish to use this later? If not though I'd delete this zombie (commented out) code